redis/src/redis/tests/resources/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# ---------------------------------------------------------------------------
+#  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+wget http://download.redis.io/releases/redis-5.0.7.tar.gz
+tar xzf redis-5.0.7.tar.gz
```

Comment:
Is this working on Windows?

Reply:
Have not tested on a windows machine. However, this is platform-indepentent according to Redis official documentation (https://redis.io/download).

Reply:
Currently running Redis on windows is not recommended (https://redislabs.com/ebook/appendix-a/a-3-installing-on-windows/a-3-1-drawbacks-of-redis-on-windows/). Also the download packages (https://github.com/dmajkic/redis/downloads) are not up-to-date. Therefore we may have to evaluate how we are going to support Redis connector on windows.

Reply:
Updated the Readme file in 1042765